charts/latest/azuredisk-csi-driver/templates/csi-azuredisk-node-windows.yaml

```diff
@@ -108,6 +108,7 @@ spec:
             - "--user-agent-suffix={{ .Values.driver.userAgentSuffix }}"
             - "--allow-empty-cloud-config={{ .Values.node.allowEmptyCloudConfig }}"
             - "--support-zone={{ .Values.node.supportZone }}"
+            - "--get-node-info-from-labels={{ .Values.windows.getNodeInfoFromLabels }}"
```

Comment:
This is the location where I was thinking that the value should be true, but I guess it depends on the setup and on having a component that can act as an intermediary to make the instance metadata calls (e.g. a container that intercepts the request and forwards it to the host binary that does it on behalf of the CSI Driver)

Reply:
if you are using helm chart install, you could add `--set windows.getNodeInfoFromLabels=true`, while it's better not change default value as true here

Reply:
sounds good, thanks